prometheus.go

```diff
@@ -29,7 +29,8 @@ type PrometheusCollector struct {
 
 func NewPrometheusCollector(metrics []*PrometheusMetric) *PrometheusCollector {
 	return &PrometheusCollector{
-		metrics: removeDuplicatedMetrics(metrics),
```

Comment:
removeDuplicatedMetrics this function will remove duplicate metrics based on Prometheus name label
so from below metrics last 2 metrics will be dropped. which is not valid.
so i think we have to think logic of removeDuplicatedMetrics.
```
aws_elb_healthy_host_count_minimum{dimension_AvailabilityZone="ap-southeast-1a",dimension_LoadBalancerName="abc",name="arn:aws:elasticloadbalancing:ap-southeast-1:abc"} 17
aws_elb_healthy_host_count_minimum{dimension_AvailabilityZone="ap-southeast-1b",dimension_LoadBalancerName="abc",name="arn:aws:elasticloadbalancing:ap-southeast-1:abc"} 17
aws_elb_healthy_host_count_minimum{dimension_AvailabilityZone="ap-southeast-1c",dimension_LoadBalancerName="abc",name="arn:aws:elasticloadbalancing:ap-southeast-1:abc"} 16```

and why do we even need that